tox.ini

```diff
@@ -19,6 +19,7 @@ setenv = QT_QPA_PLATFORM_PLUGIN_PATH={envsitepackagesdir}/PyQt5/plugins/platform
 deps =
     py==1.4.26
     pytest==2.7.0
+    pytest-qt==1.3.0
```

Comment:
Not released yet... I want to write a few more tests to make sure my tests didn't miss anything regarding `PyQt5` support. :smile: